borg/archiver.py

```diff
@@ -804,6 +801,8 @@ def __call__(self, parser, ns, values, option):
                                help="""toggle progress display while creating the archive, showing Original,
                                Compressed and Deduplicated sizes, followed by the Number of files seen
                                and the path being processed, default: %(default)s""")
+        subparser.add_argument('--changed', action='store_true', dest='changed', default=False,
+                               help="""display which files were added to the archive""")
```

Comment:
i thought we would split the changes into two sets:

* changed stuff
* unchanged stuff

... regardless of whether it is a directory, a file, a device or whatnot. right now, it seems we can only detected unchanged files, so --changed will also list unchanged directories and so on, but that's a limitation we can't work around right now.

--unchanged will list only `U` files. it should usually be used along with `--changed`.